manageiq-providers-ibm_power_hmc.gemspec

```diff
@@ -18,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency("ibm_power_hmc", [">= 0.1.0"])
+  spec.add_dependency "ibm_power_hmc", "~> 0.1"
```

Comment:
The `~> 0.1` wasn't from `rubocop -a` that was me, but we don't want to be doing `>=` or we'll pull in breaking major versions automatically